src/components/DisplayArticle.jsx

```diff
@@ -1,18 +1,46 @@
 import { useParams } from "react-router-dom";
 import { getArticle } from "../utils/api";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { increaseVotes } from "../utils/api";
 
 const DisplayArticle = () => {
+  const clickHandler = (e, increment) => {
+    setVotes((currVotes) => {
+      const newVotes = currVotes + increment;
+      return newVotes;
+    });
+    increaseVotes(article_id, increment).then((res) => {});
+  };
+
   const { article_id } = useParams();
   const [selectedArticle, setSelectedArticle] = useState({});
-  getArticle(article_id).then((article) => {
-    setSelectedArticle(article);
-  });
+  const [votes, setVotes] = useState(selectedArticle.votes);
+
+  useEffect(() => {
+    getArticle(article_id).then((article) => {
+      setSelectedArticle(article);
+    });
+  }, [votes]);
```

Comment:
As it stands we would run this useEffect every single time the vote was increased, which in turn would run the getArticle function. 

Your setVotes will cause a re-render when we invoke it. 

We will only want this useEffect to run on the first initial render, so should pass in an empty array.